library/src/scala/tasty/Reflection.scala

```diff
@@ -51,10 +51,9 @@ trait Reflection extends reflect.Types { reflectSelf: CompilerInterface =>
 
   // ----- Tree -----------------------------------------------------
 
-  given TreeOps as Tree.type = Tree
+  object Tree
 
-  object Tree:
-    /** Members of Tree */
+  given TreeOps as AnyRef:
```

Comment:
Can we remove `given TreeOps` and put extension methods directly inside `Reflection`?

Reply:
There are many methods with the same names that may clash. For example `show` is at least 5 times and to flatten these givens, we would need to add 5x `using DummyImplicit` to one of the variants.